cherab/openadas/repository/atomic.py

```diff
@@ -108,6 +108,47 @@ def update_recombination_rates(rates, repository_path=None):
         _update_and_write_adf11(species, rate_data, path)
 
 
+def add_thermalchargeexchange_rate(species, charge, rate, repository_path=None):
```

Comment:
What about something like:
```
    def add_thermal_cx_rate(self, donor_ion, donor_charge, receiver_ion, receiver_charge, rate, repository_path=None):
```

Reply:
[renamed](https://github.com/Mateasek/cherab_openadas/blob/e9458e956af735110764c7f8d3c7403889e7d9de/cherab/openadas/repository/atomic.py#L111) to:

```python
def add_thermal_cx_rate(species, charge, rate, repository_path=None):
```